### ISSUE TITLE
Make a function to work like a alarm clock for long breaks

### ISSUE THREAD

**Issue by Raghwendra-Dey** [Raghwendra-Dey/productivity_meter#28]
Write a function which would print a message to the `console.log` when the time elapsed during the long break crosses the value set in the `breakDur` variable. Also you need to repeat the message after every 5 mins.
It would be like:
* print the message first time when the elapsed time crosses `breakDur` value.
* again print it next time after 5mins, again after 5mins.. and so on. untill the user starts both the timers once again, indicating, he is at work again. Its very similar to working of `alarm clock` and its snooze feature.

This is a part of the notification system of the notification system we are trying to develop.
Comment below if you need some more clarifications.
Make all the PRs related to this in the `notification` branch of this repo.

**Comment by Raghwendra-Dey** [comment]
@TusharDasMe would you like to work on this?

**Comment by TusharDasMe** [comment]
Ok basically u need a alarm clock right?

**Comment by Raghwendra-Dey** [comment]
yeah, an alarm clock function which have snooze duration of 5mins, and just prints a message to the console. Later on we will integrate it with #27 to make a fully functional notification system.

**Comment by Raghwendra-Dey** [comment]
@TusharDasMe are you working on this?

**Comment by Raghwendra-Dey** [comment]
@dollyjain51099 , would you like to take this up?

**Comment by KerinPithawala** [comment]
Can I work on this issue?

**Comment by Raghwendra-Dey** [comment]
@KerinPithawala please go ahead.

**Comment by Raghwendra-Dey** [comment]
Hey @KerinPithawala are you facing any problems with this?
